src/data_loader.py

```diff
@@ -97,6 +118,12 @@ def delete_dataset(self, schema_id):
 
         # Clean up the access & dataset tables
         try:
+            id = schema_id[-1:]
```

Comment:
This won't work for schema names with more than a single digit id. Use `schema_id.split('-')[1]` or, if you want to remove the '-' from the schema names, use a regex or some other Python built-in function.

Reply:
my bad